src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -101,7 +102,10 @@ object WorkflowDescriptor {
     validateWorkflowDescriptor(id, sourceFiles, CromwellBackend.backend(), conf) match {
       case scalaz.Success(w) => w
       case scalaz.Failure(f) =>
-        throw new IllegalArgumentException(s"""Workflow $id failed to process inputs:\n${f.toList.mkString("\n")}""")
+        throw new IllegalArgumentException() with ThrowableWithErrors {
+          val message = s"Workflow input processing failed."
```

Comment:
Removed the ID in the error message because I think it's confusing to send back a Workflow ID if the validation failed (which means the workflow is not kicked off)